schemas/bolt-config.schema.json

```diff
@@ -170,22 +170,22 @@
       "type": "string"
     },
     "docker": {
-      "$ref": "bolt-transport-definitions.json#/docker"
+      "$ref": "https://forgeapi.puppet.com/schemas/bolt-transport-definitions.json#/docker"
```

Comment:
Can we make this a relative ref somehow so that it works with a local schema if the file is local?

Reply:
It looks like just leaving that as `bolt-transport-definitions.json#...` works. I'll update the PR to remove the changes.